app/views/NewHomeView.coffee

```diff
@@ -110,6 +110,8 @@ module.exports = class NewHomeView extends RootView
     $(window).on 'resize', @fitToPage
     @fitToPage()
     setTimeout(@fitToPage, 0)
+    if document.location.hash is '#create-account'
```

Comment:
Navigating to `/home#create-account` shows the home page and opens the create account modal. It's so that we can have a direct link to account creation.